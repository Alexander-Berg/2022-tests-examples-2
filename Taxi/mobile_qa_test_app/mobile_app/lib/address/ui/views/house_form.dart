import 'package:flutter_hooks/flutter_hooks.dart';
import 'package:flutter_txm_ui_components/components.dart';
import 'package:hooks_riverpod/hooks_riverpod.dart';
import '../../../utils/localization.dart';
import '../../address_providers.dart';

class HouseForm extends HookConsumerWidget {
  const HouseForm({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context, WidgetRef ref) {
    final pageManager = ref.watch(addressPageManager);
    final pageStateHolder = ref.watch(addressPageStateHolderProvider.notifier);
    final address = pageStateHolder.address;
    final houseInputController = useTextEditingController(text: address.house);
    final corpusInputController =
        useTextEditingController(text: address.corpus);
    final buildingInputController =
        useTextEditingController(text: address.building);

    return Row(
      children: [
        Expanded(
          child: YXListInput(
            controller: houseInputController,
            borderType: YXListBorderType.bottom,
            placeholderHasDots: false,
            placeholder: Strings.of(context).houseInputPlaceholder,
            onChanged: pageManager.onChangedHouse,
          ),
        ),
        Expanded(
          child: YXListInput(
            controller: corpusInputController,
            borderType: YXListBorderType.bottom,
            placeholderHasDots: false,
            placeholder: Strings.of(context).corpusInputPlaceholder,
            onChanged: pageManager.onChangedCorpus,
          ),
        ),
        Expanded(
          child: YXListInput(
            controller: buildingInputController,
            borderType: YXListBorderType.bottom,
            placeholderHasDots: false,
            placeholder: Strings.of(context).buildingInputPlaceholder,
            onChanged: pageManager.onChangedBuilding,
          ),
        ),
      ],
    );
  }
}
