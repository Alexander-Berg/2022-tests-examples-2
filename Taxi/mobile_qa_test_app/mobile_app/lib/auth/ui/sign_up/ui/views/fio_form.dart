// import 'package:flutter_hooks/flutter_hooks.dart';
import 'package:flutter_txm_ui_components/components.dart';
import '../../../../../utils/localization.dart';

class FioForm extends StatelessWidget {
  final ValueChanged<String>? onSurnameChanged;
  final ValueChanged<String>? onNameChanged;
  final ValueChanged<String>? onPatronymicChanged;
  const FioForm({
    Key? key,
    this.onSurnameChanged,
    this.onNameChanged,
    this.onPatronymicChanged,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) => Column(
        children: [
          YXListInput(
            hint: Strings.of(context).surnameInputHint,
            placeholderHasDots: false,
            useHintAsPlaceholder: true,
            borderType: YXListBorderType.bottom,
            onChanged: onSurnameChanged,
          ),
          YXListInput(
            borderType: YXListBorderType.bottom,
            hint: Strings.of(context).nameInputHint,
            placeholderHasDots: false,
            useHintAsPlaceholder: true,
            onChanged: onNameChanged,
          ),
          YXListInput(
            hint: Strings.of(context).patronymicInputHint,
            placeholderHasDots: false,
            useHintAsPlaceholder: true,
            borderType: YXListBorderType.bottom,
            onChanged: onPatronymicChanged,
          ),
        ],
      );
}
