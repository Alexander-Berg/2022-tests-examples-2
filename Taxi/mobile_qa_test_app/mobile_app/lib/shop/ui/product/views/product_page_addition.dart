import 'package:flutter_txm_ui_components/components.dart';
import 'package:models/models.dart';

import '../../../../utils/localization.dart';
import '../../components/shop_section.dart';

class ProductPageAddition extends StatelessWidget {
  final Product product;
  const ProductPageAddition({
    Key? key,
    required this.product,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) => ShopSection(
        child: Column(
          children: [
            YXListItem(
              title: Strings.of(context).aboutProduct,
            ),
            YXListItem(
              borderType: YXListBorderType.none,
              title: Strings.of(context).fullname,
              subtitle: product.fullName,
            ),
            if (product.parameters?.isNotEmpty == true)
              YXListItem(
                borderType: YXListBorderType.none,
                title: Strings.of(context).shortDescription,
              ),
            if (product.parameters?.isNotEmpty == true)
              ...product.parameters
                      ?.map((parameter) => YXListItem(
                            borderType: YXListBorderType.none,
                            title: parameter.name,
                            detailTitle: parameter.value,
                          ))
                      .toList() ??
                  [],
          ],
        ),
      );
}
