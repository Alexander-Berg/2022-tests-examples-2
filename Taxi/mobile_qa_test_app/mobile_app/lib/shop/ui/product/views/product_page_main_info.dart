import 'package:flutter_txm_ui_components/components.dart';
import 'package:hooks_riverpod/hooks_riverpod.dart';
import 'package:models/models.dart';

import '../../../../cart/domain/cart_state.dart';
import '../../../../cart/providers.dart';
import '../../../../common/components/txm_counter_button.dart';
import '../../../../common/components/txm_small_button.dart';
import '../../../../utils/extensions.dart';
import '../../../../utils/formatters.dart';
import '../../../../utils/localization.dart';
import '../../components/shop_section.dart';

class ProductPageMainInfo extends ConsumerWidget {
  final Product product;

  const ProductPageMainInfo({required this.product, Key? key})
      : super(key: key);

  @override
  Widget build(BuildContext context, WidgetRef ref) {
    final theme = YXTheme.of<YXThemeData>(context);
    const productImageHeight = 200.0;
    final cartItem = ref.watch(
      cartStateHolderProvider.select((s) => s.getCartItem(product)),
    );
    final cartManager = ref.watch(cartManagerProvider);

    return ShopSection(
      borderRadiusType: ShopSectionBorderRadiusType.bottom,
      child: Column(
        children: [
          TXMImageViewer(
            url: product.imageUrl,
            height: productImageHeight,
          ),
          YXListItem(
            borderType: YXListBorderType.none,
            title: product.name,
            subtitle:
                (product.count != null) ? product.amountOf(context) : null,
            subtitleColor: theme.listItemTheme.titleStyle.color,
          ),
          YXListItem(
            borderType: YXListBorderType.none,
            title: product.finalPrice.toPrice(
              currencySymbol: Strings.of(context).rubleSign,
              locale: Localizations.localeOf(context).toLanguageTag(),
              fractionDigits: 2,
            ),
            trail: (cartItem == null)
                ? TXMSmallButton(
                    isAccent: true,
                    title: Strings.of(context).addToCard,
                    onTap: () => cartManager.addProduct(product),
                  )
                : TXMCounterButton(
                    cartItem.count,
                    onDecrement: () => cartManager.removeProduct(product),
                    onIncrement: () => cartManager.addProduct(product),
                  ),
          ),
        ],
      ),
    );
  }
}
