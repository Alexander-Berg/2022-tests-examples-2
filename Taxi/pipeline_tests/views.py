import copy
import typing

from aiohttp import web

from js_pipeline.generated import models

from admin_pipeline import common
from admin_pipeline import common_helpers
from admin_pipeline.generated.service.swagger.models import api


def drop_id(doc: dict) -> dict:
    data = copy.copy(doc)
    if '_id' in doc:
        del data['_id']
    if 'id' in doc:
        del data['id']
    return data


class NotFoundError(Exception):
    ERR_CODE = 'not_found'


class _CommonView(common_helpers.DBHelper):
    DATA_TYPE: typing.Optional[typing.Type] = None

    @staticmethod
    def __for_db(data):
        return common.for_db(data, id_type=str)

    @classmethod
    async def create(cls, request, context):
        data = request.body
        existing = await cls.try_find_one(
            context.mongo.secondary,
            {'_id': data.id},
            request.consumer,
            for_model=cls.DATA_TYPE,
        )
        if existing is not None:
            raise web.HTTPBadRequest(
                reason=f'Item with id ({data.id}) already exists',
            )
        serialized = cls.__for_db(cls.DATA_TYPE.serialize(data))
        serialized['consumer'] = request.consumer
        await cls.insert_one(context.mongo, serialized)

    @classmethod
    async def get(cls, id_, consumer, context):
        found = await cls.try_find_one(
            context.mongo.secondary,
            {'_id': id_},
            consumer,
            for_model=cls.DATA_TYPE,
        )
        if not found:
            raise NotFoundError('Item not found: %s for %s' % (id_, consumer))
        return cls.DATA_TYPE.deserialize(common.for_json(found))

    @classmethod
    async def update_item(cls, request, context):
        item = request.body
        if item.id != request.id:
            raise web.HTTPBadRequest(
                reason='Item id changed! Old %s, new - %s '
                % (request.id, item.id),
            )
        doc = cls.__for_db(cls.DATA_TYPE.serialize(item))
        doc['consumer'] = request.consumer
        result = await cls.update(
            context.mongo,
            {'$and': [{'_id': request.id}, {'consumer': request.consumer}]},
            doc,
        )
        if result['n'] != 1:
            raise NotFoundError(
                'Item not found: %s for %s' % (request.id, request.consumer),
            )

    @classmethod
    async def delete(cls, request, context):
        await cls.remove(
            context.mongo,
            {'$and': [{'_id': request.id}, {'consumer': request.consumer}]},
        )


class ChecksView(_CommonView):
    DATA_TYPE = api.Check
    PROJECTIONS = {DATA_TYPE: ['name', 'check']}

    @classmethod
    def determine_check_type(cls, check):
        if 'source_code' in check:
            return 'imperative'
        if 'expected_output' in check:
            return 'combined'
        raise Exception('Unexpected check type: ' + str(check))

    @classmethod
    def get_collection(cls, db):
        return db.admin_pipeline_test_checks

    @classmethod
    async def enumerate_checks(cls, consumer, context):
        all_checks = cls.find_many(
            context.mongo, {}, consumer, for_model=api.Check,
        )
        result = api.TestChecksEnumerateResponse(
            output_checks=[
                api.EnumeratedCheck(
                    check['_id'],
                    check['name'],
                    cls.determine_check_type(check['check']),
                )
                async for check in all_checks
            ],
        )
        return result


class MocksView(_CommonView):
    DATA_TYPE = api.Mock
    PROJECTIONS = {DATA_TYPE: ['name', 'resource', 'is_prefetched', 'mock']}

    @classmethod
    def get_collection(cls, db):
        return db.admin_pipeline_test_mocks

    @classmethod
    async def enumerate_mocks(cls, consumer, context):
        all_mocks = cls.find_many(
            context.mongo, {}, consumer, for_model=api.Mock,
        )
        input_mocks = []
        prefetched_resources_mocks = []
        resources_mocks = []
        async for mock in all_mocks:
            resource = mock.get('resource')
            name = mock['name']
            _id = mock['_id']
            is_prefetched = mock['is_prefetched']
            if not resource:
                input_mocks.append(api.EnumeratedMocksItem(_id, name))
            elif is_prefetched:
                prefetched_resources_mocks.append(
                    api.EnumeratedMocksItem(_id, name, resource),
                )
            else:
                resources_mocks.append(
                    api.EnumeratedMocksItem(_id, name, resource),
                )

        result = api.TestMocksEnumerateResponse(
            input_mocks=input_mocks,
            resources_mocks=resources_mocks,
            prefetched_resources_mocks=prefetched_resources_mocks,
        )
        return result


class TestsView(_CommonView):
    DATA_TYPE = models.PipelineTest
    PROJECTIONS = {
        DATA_TYPE: [
            'name',
            'scope',
            'testcases',
            'prefetched_resources_mocks',
            'resources_mocks',
            'input_mocks',
            'output_checks',
        ],
    }

    @classmethod
    def get_collection(cls, db):
        return db.admin_pipeline_tests

    @classmethod
    async def enumerate_tests(cls, request, context):
        all_tests = cls.find_many(
            context.mongo, {}, request.consumer, for_model=models.PipelineTest,
        )
        result = api.TestsEnumerateResponse(
            tests=[
                api.EnumeratedTest(test['_id'], test['name'], test['scope'])
                async for test in all_tests
            ],
        )
        return result


class ChoosenTestsHelper(common_helpers.DBHelper):
    PROJECTIONS: dict = dict()

    @classmethod
    def get_collection(cls, db):
        return db.admin_pipeline_choosen_tests


class TestsResultsHelper(common_helpers.DBHelper):
    PROJECTIONS: dict = {models.PipelineTestsResults: ['tests', 'created']}

    @classmethod
    def get_collection(cls, db):
        return db.admin_pipeline_tests_results
