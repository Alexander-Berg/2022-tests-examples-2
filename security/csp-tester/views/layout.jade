doctype html
html
  head
    title= title
    link(rel='stylesheet', href='/stylesheets/style.css')
    link(rel='stylesheet', href='/stylesheets/highlight/styles/tomorrow.css')
    script(src='/javascripts/highlight.pack.js')
  body
    h1 CSP Tester
    script hljs.initHighlightingOnLoad();
    block content
