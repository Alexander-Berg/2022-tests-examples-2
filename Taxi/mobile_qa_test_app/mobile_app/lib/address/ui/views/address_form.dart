import 'package:flutter_hooks/flutter_hooks.dart';
import 'package:flutter_txm_ui_components/components.dart';
import 'package:hooks_riverpod/hooks_riverpod.dart';

import '../../../auth/domain/user_state_holder.dart';
import '../../../utils/localization.dart';
import '../../address_providers.dart';
import 'house_form.dart';

class AddressForm extends HookConsumerWidget {
  const AddressForm({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context, WidgetRef ref) {
    final pageManager = ref.watch(addressPageManager);
    final address = ref.watch(userStateProvider.select((u) => u?.address));
    final cityInputController = useTextEditingController(text: address?.city);
    final streetInputController =
        useTextEditingController(text: address?.street);

    return Column(
      children: [
        YXListInput(
          controller: cityInputController,
          borderType: YXListBorderType.bottom,
          placeholderHasDots: false,
          placeholder: Strings.of(context).cityInputPlaceholder,
          onChanged: pageManager.onChangedCity,
        ),
        YXListInput(
          controller: streetInputController,
          borderType: YXListBorderType.bottom,
          placeholderHasDots: false,
          placeholder: Strings.of(context).streetInputPlaceholder,
          onChanged: pageManager.onChangedStreet,
        ),
        const HouseForm(),
      ],
    );
  }
}
