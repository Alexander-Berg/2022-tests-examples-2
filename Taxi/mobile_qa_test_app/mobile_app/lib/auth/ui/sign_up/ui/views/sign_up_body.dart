import 'package:flutter_txm_ui_components/components.dart';
import '../../../../../utils/localization.dart';

import 'fio_form.dart';

class SignUpBody extends StatelessWidget {
  final ValueChanged<String>? onSurnameChanged;
  final ValueChanged<String>? onNameChanged;
  final ValueChanged<String>? onPatronymicChanged;
  const SignUpBody({
    Key? key,
    this.onSurnameChanged,
    this.onNameChanged,
    this.onPatronymicChanged,
  }) : super(key: key);

  @override
  Widget build(BuildContext context) => Column(
        mainAxisAlignment: MainAxisAlignment.spaceBetween,
        children: [
          YXListText(
            text: Strings.of(context).buildBody,
            borderType: YXListBorderType.none,
          ),
          FioForm(
            onSurnameChanged: onSurnameChanged,
            onNameChanged: onNameChanged,
            onPatronymicChanged: onPatronymicChanged,
          ),
        ],
      );
}
