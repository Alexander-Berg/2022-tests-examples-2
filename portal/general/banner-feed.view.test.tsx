import { execView } from '@lib/views/execView';
import { mockReq } from '@lib/views/mockReq';

describe('banner-feed', function() {
    let req = mockReq({}, {
        BannerFeed: {
            data: {
                banner_feed_1: {
                    height: 300,
                    'imp-id': 'N-I-534228-3'
                },
                banner_feed_2: {
                    height: 201,
                    'imp-id': 'N-I-534228-33',
                },
                banner_feed_3: {
                    height: 201,
                    'imp-id': 'N-I-534228-33',
                },
                banner_feed_4: {
                    height: 201,
                    'imp-id': 'N-I-534228-33',
                }
            },
            processed: 1,
            show: 1
        },
        blocks_layout_ext: [
            {
                id: 'directfeed',
                layout_item: {
                    footer: 'Все предложения',
                    footer_url: 'https://yandex.ru/search/touch/direct/?from=home&source=yandex_portal&vcards=1&vhome=1&text=Объявления',
                    icon: 'direct0',
                    id: 'directfeed',
                    is_adv: '1',
                    subtitle: 'Персональные предложения',
                    title: 'Яндекс.Директ',
                    title_url: '//direct.yandex.ru/?partner'
                }
            },
            {
                id: 'directfeed_2',
                layout_item: {
                    footer: 'Все предложения',
                    footer_url: 'https://yandex.ru/search/touch/direct/?from=home&source=yandex_portal&vcards=1&vhome=1&text=Объявления',
                    icon: 'direct0',
                    id: 'directfeed_2',
                    is_adv: '1',
                    subtitle: 'Персональные предложения'
                }
            },
            {
                id: 'directfeed_3',
                layout_item: {
                    icon: 'direct0',
                    id: 'directfeed_3',
                    is_adv: '1',
                    subtitle: 'Персональные предложения'
                }
            },
            {
                id: 'directfeed_4',
                layout_item: {
                    footer: 'Все предложения',
                    footer_url: 'https://yandex.ru/search/touch/direct/?from=home&source=yandex_portal&vcards=1&vhome=1&text=Объявления',
                    id: 'directfeed_4',
                    is_adv: '1',
                    subtitle: 'Персональные предложения',
                    title: 'Яндекс.Директ',
                    title_url: '//direct.yandex.ru/?partner'
                }
            }
        ],
        JSON: {
            common: {
                pageName: 'index'
            }
        },
        options: {}
    });

    it('banner-feed_1', function() {
        expect(execView('banner-feed_1', {}, req)).toMatchSnapshot();
    });

    it('banner-feed_2', function() {
        expect(execView('banner-feed_2', {}, req)).toMatchSnapshot();
    });

    it('banner-feed_3', function() {
        expect(execView('banner-feed_3', {}, req)).toMatchSnapshot();
    });

    it('banner-feed_4', function() {
        expect(execView('banner-feed_4', {}, req)).toMatchSnapshot();
    });
});
