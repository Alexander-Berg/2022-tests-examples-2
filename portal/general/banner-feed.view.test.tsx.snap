// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`banner-feed banner-feed_1 1`] = `"<div  class=\\"block banner-feed banner-feed_place_1 block_viewed_no banner-feed_type_default\\"  data-bem=\\"{&quot;banner-feed&quot;:{&quot;impId&quot;:&quot;N-I-534228-3&quot;,&quot;className&quot;:&quot;banner-feed_place_1&quot;,&quot;contextNewUrl&quot;:false,&quot;autoruExp&quot;:false}}\\" data-cls=\\"banner-feed_place_1\\"><h1 class='block__title'><a class=\\"home-link banner-feed__title block__title-text block__title-text_icon_yes block__title-text_rich_yes home-link_black_yes\\" href=\\"//direct.yandex.ru/?partner\\" data-statlog=\\"directfeed.title\\"><span  class=\\" block__title-icon\\" style=\\"background-image:url(//yastatic.net/s3/home/feed/services/icons/direct0.svg)\\"></span>Яндекс.Директ<span class='block__subtitle'>Персональные предложения</span></a></h1><div  class=\\"banner-feed__content block__content\\"><style  >.banner-feed_place_1 .banner-feed__layout{height:300px;}</style><div  class=\\"banner-feed__layout\\" id=\\"banner-feed_place_1\\"></div></div><div  class=\\"block__footer-wrapper\\"><a class=\\"home-link block__footer home-link_black_yes\\" href=\\"https://yandex.ru/search/touch/direct/?from=home&source=yandex_portal&vcards=1&vhome=1&text=Объявления\\" data-statlog=\\"banner_feed_1.footer\\">Все предложения</a></div></div>"`;

exports[`banner-feed banner-feed_2 1`] = `"<div  class=\\"block banner-feed banner-feed_place_2 block_viewed_no banner-feed_type_default\\"  data-bem=\\"{&quot;banner-feed&quot;:{&quot;impId&quot;:&quot;N-I-534228-33&quot;,&quot;className&quot;:&quot;banner-feed_place_2&quot;,&quot;contextNewUrl&quot;:false,&quot;autoruExp&quot;:false}}\\" data-cls=\\"banner-feed_place_2\\"><div  class=\\"banner-feed__content block__content\\"><style  >.banner-feed_place_2 .banner-feed__layout{height:201px;}</style><div  class=\\"banner-feed__layout\\" id=\\"banner-feed_place_2\\"></div></div><div  class=\\"block__footer-wrapper\\"><a class=\\"home-link block__footer home-link_black_yes\\" href=\\"https://yandex.ru/search/touch/direct/?from=home&source=yandex_portal&vcards=1&vhome=1&text=Объявления\\" data-statlog=\\"banner_feed_2.footer\\">Все предложения</a></div></div>"`;

exports[`banner-feed banner-feed_3 1`] = `"<div  class=\\"block block_footer_no banner-feed banner-feed_place_3 block_viewed_no banner-feed_type_default\\"  data-bem=\\"{&quot;banner-feed&quot;:{&quot;impId&quot;:&quot;N-I-534228-33&quot;,&quot;className&quot;:&quot;banner-feed_place_3&quot;,&quot;contextNewUrl&quot;:false,&quot;autoruExp&quot;:false}}\\" data-cls=\\"banner-feed_place_3\\"><div  class=\\"banner-feed__content block__content\\"><style  >.banner-feed_place_3 .banner-feed__layout{height:201px;}</style><div  class=\\"banner-feed__layout\\" id=\\"banner-feed_place_3\\"></div></div></div>"`;

exports[`banner-feed banner-feed_4 1`] = `"<div  class=\\"block banner-feed banner-feed_place_4 block_viewed_no banner-feed_type_default\\"  data-bem=\\"{&quot;banner-feed&quot;:{&quot;impId&quot;:&quot;N-I-534228-33&quot;,&quot;className&quot;:&quot;banner-feed_place_4&quot;,&quot;contextNewUrl&quot;:false,&quot;autoruExp&quot;:false}}\\" data-cls=\\"banner-feed_place_4\\"><h1 class='block__title'><a class=\\"home-link banner-feed__title block__title-text block__title-text_rich_yes home-link_black_yes\\" href=\\"//direct.yandex.ru/?partner\\" data-statlog=\\"directfeed_4.title\\">Яндекс.Директ<span class='block__subtitle'>Персональные предложения</span></a></h1><div  class=\\"banner-feed__content block__content\\"><style  >.banner-feed_place_4 .banner-feed__layout{height:201px;}</style><div  class=\\"banner-feed__layout\\" id=\\"banner-feed_place_4\\"></div></div><div  class=\\"block__footer-wrapper\\"><a class=\\"home-link block__footer home-link_black_yes\\" href=\\"https://yandex.ru/search/touch/direct/?from=home&source=yandex_portal&vcards=1&vhome=1&text=Объявления\\" data-statlog=\\"banner_feed_4.footer\\">Все предложения</a></div></div>"`;
